bench_test.go

```diff
@@ -7,7 +7,7 @@ import (
 	"testing"
 	"time"
 
-	"github.com/go-redis/redis"
+	"github.com/go-redis/redis/v6"
```

Comment:
Do we really must specify version `v6` in tests? It is a bit strange that module should "versionify" itself.